.travis.yml

```diff
@@ -6,13 +6,19 @@ script: "bundle exec rake validate && bundle exec rake lint && bundle exec rake
 matrix:
   fast_finish: true
   include:
+  - rvm: 1.9.3
+    env: PUPPET_GEM_VERSION="~> 3.4.0"
   - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 2.7.0" FACTER_GEM_VERSION="~> 1.6.0"
-  - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 2.7.0" FACTER_GEM_VERSION="~> 1.7.0"
+    env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 1.9.3
     env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 2.0.0
```

Comment:
Why are we removing 2.0 we still use this explicitly with PE 3.7 on Windows. There is enough difference with 2.1.5 and 2.0 that I would rather see both